minidump/src/minidump.rs

```diff
@@ -2626,6 +2630,51 @@ impl MinidumpSystemInfo {
     pub fn cpu_info(&self) -> Option<Cow<str>> {
         self.cpu_info.as_deref().map(Cow::Borrowed)
     }
+
+    /// Strings identifying the version and build number of the operating
+    /// system.
+    ///
+    /// Tries to parse the version number from the build if it cannot be found
+    /// in the version string. If the stream already contains a valid version
+    /// number or parsing from the build string fails, this will return what's
+    /// directly stored in the stream.
+    pub fn os_parts(&self) -> (String, Option<String>) {
```

Comment:
nit reading just the doc comment and signature i have no idea what the parts in the return type tuple are.

Reply:
"Returns a tuple of the version and optional build number of the operating system" might help me.